services/analyzer/sequelize-tables-analyzer.js

```diff
@@ -139,6 +140,9 @@ function createReference(tableName, association, foreignKey, manyToManyForeignKe
     reference.ref = manyToManyForeignKey.foreignTableName;
     reference.otherKey = manyToManyForeignKey.columnName;
     reference.junctionTable = foreignKey.tableName;
```

Comment:
Remove this variable if it's not used anywhere else

Reply:
👍